mayastor/src/bdev/nexus/nexus_config.rs

```diff
@@ -103,7 +103,12 @@ pub(crate) fn parse_ini_config_file() -> i32 {
             name, lu_size, block_size, &child_bdevs
         );
 
-        nexus_instance_new(name, lu_size, child_bdevs);
+        futures::executor::block_on(nexus_instance_new(
```

Comment:
Is this "ini" code still relevant since we have the config file? Otherwise might as well remove this altogether